samcli/hook_packages/terraform/hooks/prepare/makefile.py

```diff
@@ -0,0 +1,260 @@
+"""
```

Comment:
Lets change the name of this file to be a bit more specific, something like `makefile_generator.py` or `makefile_generation.py`